app/components/moderation-list-row/component.js

```diff
@@ -44,10 +44,20 @@ export default Ember.Component.extend({
         return moment().diff(this.get('submission.dateLastTransitioned'), 'days') > 1;
     }),
 
+    fewSeconds: Ember.computed('submission.dateLastTransitioned', function() {
+        const timeDiff = Math.abs(moment().diff(this.get('submission.dateLastTransitioned')));
+        if (timeDiff < 70000) { // 70000 milliseconds
```

Comment:
Is there a reason for the magic 70000?

Reply:
I used absolute value so it can handle past or future differences. Sometimes datelasttransitioned seems slightly in past too.

Reply:
Slightly in the past is fine -- what we'd expect, actually. But if it's somehow more than 70 seconds in the future, it might display like "a minute from now", which is bad.

Reply:
7000 seems to cover the differences after several trials, which slight more than a minute

Reply:
I think "a minute ago" is fine, it's close enough to true. The bug is when it displays "in a few seconds", or anything that implies the transition is in the future.

Reply:
ok 👍